lib/utils/isAutoprefixable.mjs

```diff
@@ -21,12 +21,13 @@ import vendor from './vendor.mjs';
 const AT_RULES = new Set([
 	'@-khtml-keyframes',
 	'@-moz-keyframes',
+	'@-moz-document',
 	'@-ms-keyframes',
 	'@-ms-viewport',
 	'@-o-keyframes',
 	'@-o-viewport',
 	'@-webkit-keyframes',
-	'@resolution',
```

Comment:
[ask] This `@resolution` is output from the `@example` code. I'm not sure why this unprefixed rule is included. Can you confirm autoprefixer, just in case?

Reply:
I have already put a link that goes directly to the lines in their codebase.

Reply:
Honestly, I'm not sure about `@resolution`, but this PR's changes will be away from the code generation by the `@example` scripts:

- https://github.com/stylelint/stylelint/blob/d273483ac942380d98c28b591d7005e613d0776e/lib/utils/isAutoprefixable.mjs#L8-L14
- https://github.com/stylelint/stylelint/blob/d273483ac942380d98c28b591d7005e613d0776e/lib/utils/isAutoprefixable.mjs#L18-L19

I worry that we might be confused about the difference between this PR's changes and output from `@example` scripts in the future.

So, I suggest leaving some comments to clarify this code change intention like this:

```js
AT_RULES.delete("@-khtml-keyframes"); // <some reason>
AT_RULES.delete("@resolution"); // <some reason>
AT_RULES.add("@-moz-document"); // <some reason>
```

Of course, I understand this way is a bit ugly, so I welcome a better idea if any.

Reply:
I think we should do away with the examples and replace them with an explanation of the util in its entirety placed [before the `export`](https://github.com/stylelint/stylelint/blob/main/lib/utils/isAutoprefixable.mjs#L306-L309).
For now what matters is the code itself not that we weren't careful enough during #5312.
I would like for it to be a separate documentation PR though.

Reply:
Agree. It seems we need to improve the document in this file. 👍🏼